scripts/image-builder/configure.sh

```diff
@@ -17,6 +17,9 @@ sudo dnf install -y "https://dl.fedoraproject.org/pub/epel/epel-release-latest-$
 sudo dnf install -y mock caddy tomcli
 sudo usermod -a -G mock "$(whoami)"
 
+# Set the Correct Permissions for osbuild-composer
+sudo chmod 0755 /home/microshift
```

Comment:
Does the whole home directory actually need to be world readable or can it be executable so that subdirectories can be accessed but someone can't read my dot files?

Reply:
@eslutsky , can you test if osbuilder works in the following configuration?
```
chmod 700 ~microshift
chmod ugo+x ~microshift
```

Reply:
Also, since we're testing HOME_PERM below, let's only change the permissions is it's necessary - something like the following:
```
[ "${HOME_PERM}" -lt 0755 ]  && chmod 755 ~
if [ "${FILE_PERM}" -lt 0644 ] || [ "${DIR_PERM}" -lt 0755 ] ; then
    echo "Check your umask. The settings must allow read to group and others"
    exit 1
fi
```

Reply:
> Does the whole home directory actually need to be world readable or can it be executable so that subdirectories can be accessed but someone can't read my dot files?
@dhellmann , i agree that  ch-modding 700 to  home dir is too permissive and not  a good idea , 
@ggiguash   `+x`  does solve our problem and its  much safer.